src/PowerSystems2PRAS.jl

```diff
@@ -862,6 +886,65 @@ function generate_pras_system(
     end
 end
 
+const DEFAULT_DEVICE_MODELS = [
+    DevicePRASModel(PSY.ThermalGen, GeneratorFormulation),
+    DevicePRASModel(PSY.RenewableGen, GeneratorFormulation),
+    DevicePRASModel(PSY.HydroDispatch, GeneratorFormulation),
+    DevicePRASModel(PSY.Storage, StorageFormulation),
```

Comment:
I think this is okay but can we change PSY.Storage to PSY.EnergyReservoirStorage because we use subtypes for all other types in the the default template. Also, process_storages is formatted for PSY.EnergyReservoirStorage

Reply:
Done. This ended up being a little more different than I expected. I added a StoragePRAS formulation layer + an interface for the energy reservoir formulation.